### DISCUSSION TITLE
Can't use app but auth token normally copied from browser

### DISCUSSION THREAD

**Discussion by Bogdan-Torkhov** [masterking32/MasterHamsterKombatBot#88] [Q&A] [answered]
[Master HamsterKombat Bot][ERROR] [Account 1] Status code is not 200
[Master HamsterKombat Bot][ERROR] [Account 1] Response: {"error_code":"BAD_AUTH_TOKEN","error_message":"Bad auth header authorization=17222824769**************"}
[Master HamsterKombat Bot][ERROR] [Account 1] Unable to get account basic data.
[Master HamsterKombat Bot][ERROR] Sleeping for 56 seconds because of random delay...
[Master HamsterKombat Bot][ERROR] Rechecking all accounts in 300 seconds...

**Comment by Microzibert** [comment, marked as answer]
You have forgot to write Bearer
```python
"Authorization": "Bearer 1234567890"
```
